### ISSUE TITLE
Staging Review finding: Descriptive alt text for images

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98539]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Authenticated Experience Design Patterns (AEDP)
**Product:** Authenticated Design Patterns
**Feature:** Help users to...Update prefilled information

## Findings details
**VA.gov Experience Standard - issue:** [User can perceive all meaningful elements.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Before publishing, ensure that all images have alt text that describes their content appropriately for their context.

For example, the first example image under "Informational text after a set of uneditable information" currently has alt text:
> How to edit personal information

That doesn't really describe the image in a useful way. But something like this would:
> Note explaining why online changes for certain types of personal information are not allowed, next steps for making changes via phone, and a link to more detailed instructions that opens in a new tab.

It doesn't have to be a word-for-word description of the screenshot, but it does need to describe what's happening and why it matters to what your audience is doing.

## Recommended action

It's never too early to write alt text! Definitely not something to leave to the end.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** [Know how their information is updated](https://design.va.gov/patterns/help-users-to/know-how-their-information-is-updated)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by msbtterswrth** [comment]
@kristen101606 can you update our alt text? If you need help just lmk :)

**Comment by msbtterswrth** [comment]
This has been updated in [our guidance](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Patterns/update-prefilled-information/Help%20users%20to%20update%20prefilled%20information.md), a PR will follow once all the content has been updated.
